.github/ISSUE_TEMPLATE/5_KIT_template.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Propose KIT
+labels: kit
+---
+
+# Overview
+
+## Description
+<!-- Please provide a short description -->
+
+## Impact
+<!-- Please name the impact -->
```

Comment:
Could you give some insights on what you mean with Impact. I would assume, as a KIT is a kind of documentation, the implementation and standards have an impact on the KIT but I have no idea which impact a KIT has on something else.

Reply:
Instead of impact I could consider a point like major changes or major adjustments as a point to describe